### PR TITLE
Added managed script for under min isr partitions

### PR DESCRIPTION
**Pull request by robshelly**
This is a managed script for the under min ISR alert in Kafka. It's pretty much a copy paste of the [under replicated partition managed script](https://github.com/openshift/managed-scripts/tree/main/scripts/CSSRE/under-replicated-partitions) as the kafka.py class already has the logic to handle partitions that are under minimum ISR.

### CODE REVIEW

**Comment by robshelly** [comment]
@pepedocs, @vinaybommana, @pat-cremin can you review please?

File: scripts/CSSRE/under-min-isr-partitions/under-min-isr-partitions.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  def create_parser(self):
+    self.parser = argparse.ArgumentParser(description="Checks for Kafka under min ISR partitions.")
+    self.parser.add_argument("--kafka-namespace", help="The namespace of the kafka")
+    self.parser.add_argument("--log-level", help="Set logging level.")
+
+  def run(self):
+    self.logger.debug("---Running managed script: Under Min ISR Partitions---")
+
+    kafka = Kafka(self.oc, self.settings, logger_name="Under Min ISR Partitions")
+    kafka.check_namepace_managed_kafka(self.KAFKA_NAMESPACE)
```
Comment (pepedocs) [inline scripts/CSSRE/under-min-isr-partitions/under-min-isr-partitions.py:24]: I just saw this typo. Probably not for this PR? But just worth noting it.

**Comment by robshelly** [comment]
@pepedocs Thanks for poiting out that typo. I've fixed it now, can you approve again please?

**Review by pepedocs — APPROVED** [review]

**Comment by robshelly** [comment]
@wanghaoran1988 Can we get this approved/merged please?

**Pull request merged**